lib/src/google_fonts_base.dart

```diff
@@ -120,31 +119,50 @@ Future<void> loadFontIfNecessary(GoogleFontsDescriptor descriptor) async {
     return;
   }
 
-  // If this font can be loaded by the pre-bundled assets, then there is no
-  // need to load it at all.
-  final assetManifestJson = await _loadAssetManifestJson();
+  Future<ByteData> byteData;
 
-  if (_isFamilyWithVariantInManifest(
-    descriptor.familyWithVariant,
-    assetManifestJson,
-  )) {
-    return;
-  }
+  try {
+    // Check if this font can be loaded by the pre-bundled assets.
+    final assetManifestJson = await _loadAssetManifestJson();
+    final assetPath = _findFamilyWithVariantAssetPath(
+      descriptor.familyWithVariant,
+      assetManifestJson,
+    );
+    if (assetPath != '') {
```

Comment:
should we use null here instead of empty string as sentinel value?

Reply:
I like that better!